### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by karan-pharate**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+body  {
+background-color: aqua;
+}
+h1   {
+color: red;
+}
+p    {
+color: blue;
+}
```
Comment (Shrikant9) [inline style.css]: Add configuration to your editor so it automatically adds new line at the end of file.
If you're using vscode, add `"files.insertFinalNewline": true` to the configuration.

**Review by Shrikant9 — CHANGES_REQUESTED** [review]

File: websiteLayout/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo-container"><img src="./images/logo.jpg" alt="pizzaLogo"></div>
+        <div class="header-content">
+            <ul>
+                <li><a href="">Offers</a></li>
```
Comment (visha15l) [inline websiteLayout/index.html]: use href="javascript:;"

File: cssGrid/style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.grid-container {
```
Comment (visha15l) [inline cssGrid/style.css:1]: use prettier

File: css-assignments/position.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    .relative1{
+        position: relative;
+        width: 400px;
+        height: 200px;
+        border: 3px solid red;
+    }
+    .absolute {
+        position: absolute;
+        top: 60px;
+        right: 0px;
```
Comment (visha15l) [inline css-assignments/position.html]: write 0 instead of 0px

**Review by visha15l — APPROVED** [review]

**Pull request merged**